asset-transfer-basic/application-gateway-java/src/main/java/application/java/App.java

```diff
@@ -0,0 +1,251 @@
+                    .signer(newSigner())
+                    .connection(channel)
+                    //default timeouts for different GRPC calls
+                    .evaluateOptions(CallOption.deadlineAfter(5, TimeUnit.SECONDS))
+                    .endorseOptions(CallOption.deadlineAfter(15, TimeUnit.SECONDS))
+                    .submitOptions(CallOption.deadlineAfter(5, TimeUnit.SECONDS))
+                    .commitStatusOptions(CallOption.deadlineAfter(1, TimeUnit.SECONDS));
+               
+        try (Gateway gateway = builder.connect()) {
+        		Network network = gateway.getNetwork(channelName);
+        		Contract contract = network.getContract(chaincodeName);
+        	
+        			//Initialize the Ledger with some data
+        			System.out.println("InitLedger");
+        			initLedger(contract);
+        			System.out.println();
+                    
+                    //getAllAssets
+                    System.out.println("Get All Assets");
+                    getAllAssets(contract);
+                    System.out.println();
+                    //createAsset
+
+                    System.out.println("Create Asset");
+                    createAsset(contract);
+                    System.out.println();
+
+                    //transferAssetAsync
+                    System.out.println("Transfer Asset asynchronously");
+                    transferAssetAsync(contract);
+                    System.out.println();
+
+                    //readAssetByID
+                    System.out.println("Read Asset By ID");
+                    readAssetById(contract);
+                    System.out.println();                    
+
+                    //updateNonExistentAsset
+                    System.out.println("Update a non-existent Asset");
+                    updateNonExistentAsset(contract);
+                    System.out.println();                    
+
+                    
+
+		} catch(Throwable e){
+		  e.printStackTrace();
+		} finally {
+                    channel.shutdownNow().awaitTermination(5, TimeUnit.SECONDS);
+
+                }
+                System.exit(0);
+
+	}
+	private static void initLedger(Contract contract) throws GatewayException, CommitException {
+        
+
+        String timestamp = LocalDateTime.now().toString();
+
+        System.out.println("Submitting \"InitLedger\" transaction with arguments: time, " + timestamp);
+
+        // Submit a transaction, blocking until the transaction has been committed on the ledger.
+        byte[] submitResult = contract.submitTransaction("InitLedger", "time", timestamp);
+
+        System.out.println("Submit result: " + new String(submitResult, StandardCharsets.UTF_8));
+        System.out.println("Evaluating \"InitLedger\" query with arguments: time");
+
+        byte[] evaluateResult = contract.evaluateTransaction("InitLedger", "time");
+        System.out.println("Query result: " + new String(evaluateResult, StandardCharsets.UTF_8));
+    }
```

Comment:
This does not match the behaviour or console output of the other language samples. They need to behave identically so the [tutorial documentation](https://hyperledger-fabric.readthedocs.io/en/latest/write_first_app.html#fifth-invoke-transaction-functions-to-read-and-write-assets) is correct for all the different language versions of the same sample.

https://github.com/hyperledger/fabric-samples/blob/0d7fa171d146a59dd72bd9e52f2e6e0f8a15fc47/asset-transfer-basic/application-gateway-typescript/src/app.ts#L103-L109

Reply:
this is done